PocketKit/Sources/PocketKit/MyList/SortFilterMenu/SortMenuViewModel.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+import Sync
+import Analytics
+
+
+class SortMenuViewModel {
+
+    typealias Snapshot = NSDiffableDataSourceSnapshot<SortSection, SortOption>
+    @Published
+    private var _snapshot = Snapshot()
+    var snapshot: Published<Snapshot>.Publisher { $_snapshot }
+
+    private let source: Source
+    private let tracker: Tracker
+    private let listOptions: ListOptions
+
+    init(source: Source, tracker: Tracker, listOptions: ListOptions) {
+        self.source = source
+        self.tracker = tracker
+        self.listOptions = listOptions
+        configureDiffable()
+    }
+
+    func configureDiffable() {
```

Comment:
For the sake of consistency with other view models, can we rename this method to `buildSnapshot`?